documents/API.md

```diff
@@ -4,38 +4,43 @@ This is a draft of the operations available to meet the initial views of the app
 
 ##Operations
 
-1. Get list of users. If permission_repo is specified, the "permission" flag on each user will be present, reporting their access to that repo.
+1. Get identity of authenticated user
+  
+        GET /identity
+        returns Identity{}
+
+2. Get list of users. If permission_repo is specified, the "permission" flag on each user will be present, reporting their access to that repo.
```

Comment:
You can use '1.' repeatedly in markdown and it will auto-increment the numbers in the list.